tfx/examples/bert_cola/bert_models.py

```diff
@@ -0,0 +1,55 @@
+import tensorflow.keras as keras
+import tensorflow as tf
+
+def BertForSingleSentenceSentimentAnalysis(
+    bert_layer,
+    max_len,
+    hidden_layers=[]):
+    """Keras model for single sentence sentiment analysis. Connect
+    configurable fully connected layers on top of the Bert pooled_output.
+
+    Args:
+        bert_layer: A tensroflow_hub.KerasLayer intence of Bert layer.
+        max_len: The maximum length of preprocessed tokens.
+        hidden_layers: List of configurations for fine-tuning hidden layers
+            after the pooled_output. [(#of hidden units, activation)].
+
+    Returns:
+        A Keras model.
+    """
+    input_id_layer = keras.layers.Input(
+        shape=(max_len,),
+        dtype=tf.int32,
+        name="input_word_ids")
+    
+    input_mask_layer = keras.layers.Input(
+        shape=(max_len,),
+        dtype=tf.int32,
+        name="input_mask")
+    
+    input_type_ids_layer = keras.layers.Input(
+        shape=(max_len,),
+        dtype=tf.int32,
+        name="segment_ids")
+
+    pooled_output, sequence_output = bert_layer([
+        input_id_layer,
+        input_mask_layer,
+        input_type_ids_layer])
+
+    hidden = pooled_output
```

Comment:
Should we be using the sequence output corresponding to the first token (CLS) or pooled_output?

Reply:
seems like pooled_output

Reply:
yes, lets write a comment saying that the pooled input for this model is a dense layer on top of the CLS token so that others know about the rationale.